docs/DOCKER.md

```diff
@@ -65,3 +65,30 @@ $ docker logs lnd-testnet
 
 This is a simple example, it is possible to use any command-line options necessary
 to expose RPC ports, use `btcd` or `bitcoind`, or add additional chains.
+
+## LND Development and Testing
+
+To test the Docker production image locally, run the following from
+the project root:
+
+```
+$ docker build . -t lnd:1.0
+```
+
+To choose a specific branch or tag instead, use the "checkout" build-arg.  For example, to build the latest commits in master:
+
+```
+$ docker build . --build-arg checkout=master -t lnd:1.0
```

Comment:
This will result in the exact same image as the example above. Why not give an example of a production tag (e.g. `v0.8.0-beta`) and then also name the image `lnd:0.8.0-beta`?

Reply:
adding tag as an example is a good idea.  I don't have a problem adding that.